packages/wrangler/src/ai/fetcher.ts

```diff
@@ -16,12 +16,14 @@ export default function (env) {
 export async function AIFetcher(request: Request): Promise<Response> {
 	const accountId = await getAccountId();
 
-	request.headers.delete("Host");
-	request.headers.delete("Content-Length");
+	const reqHeaders = new Headers(request.headers);
+	reqHeaders.delete("Host");
+	reqHeaders.delete("Content-Length");
+	reqHeaders.set("x-url", request.url);
```

Comment:
I think there is a standard one can use for proxying requests like this...

Reply:
Oh I was thinking of https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-Host